tests/rptest/tests/archival_test.py

```diff
@@ -224,7 +224,7 @@ def __init__(self, test_context):
 
         self._s3_port = self.si_settings.cloud_storage_api_endpoint_port
 
-        self.kafka_tools = KafkaCliTools(self.redpanda)
+        self.kafka_tools = KafkaCliTools(self.redpanda, version="3.9.0")
```

Comment:
This is currently default version, can you describe the intent of this change in the commit message, please